app/backend/prepdocs.py

```diff
@@ -154,10 +156,9 @@ def setup_file_processors(
     local_html_parser: bool = False,
     search_images: bool = False,
 ):
-    html_parser: Parser
-    pdf_parser: Parser
-    doc_int_parser: DocumentAnalysisParser
+    sentence_text_splitter = SentenceTextSplitter(has_image_embeddings=search_images)
```

Comment:
I changed this code around a bit as I found some issues once I wrote tests that touched it.